quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/ls/QuarkusTextDocumentService.java

```diff
@@ -143,6 +147,20 @@ public void didSave(DidSaveTextDocumentParams params) {
 		});
 	}
 
+	@Override
+	public CompletableFuture<List<Either<SymbolInformation, DocumentSymbol>>> documentSymbol(
+			DocumentSymbolParams params) {
+		return getPropertiesModel(params.getTextDocument(), (cancelChecker, document) -> {
+			return getQuarkusLanguageService().findSymbolInformations(document, cancelChecker) //
+					.stream() //
+					.map(s -> {
```

Comment:
.map(Either::forLeft)

Reply:
I tried it to do that, but I had compilation problem?